static_report/src/components/shared/Sidebar.tsx

```diff
@@ -0,0 +1,171 @@
+  useDisclosure,
+  useToast,
+} from '@chakra-ui/react';
+import { FiDatabase, FiMessageSquare } from 'react-icons/fi';
+import { useState } from 'react';
+
+type Feedback = {
+  user_id: string;
+  message: string;
+  version?: string;
+  score?: number;
+  page?: string;
+  misc?: string;
+};
+
+async function sendFeedback(data: Feedback): Promise<boolean> {
+  try {
+    await fetch(
+      `https://api.aws.piperider.io/v1/feedback?token=371aa67a-a6eb-44cc-9d02-4a05810f24ac`,
+      {
+        method: 'POST',
+        body: JSON.stringify(data),
+        headers: new Headers({
+          'Content-Type': 'application/json',
+        }),
+      },
+    ).then((res) => res.json());
+
+    return true;
+  } catch (err) {
+    console.error(err);
+    return false;
+  }
+}
+
+export function Sidebar() {
+  const toast = useToast();
+  const modal = useDisclosure();
+  const [feedback, setFeedback] = useState('');
+  const [isSending, setIsSending] = useState(false);
+
+  return (
+    <Flex
+      height="100vh"
+      position="sticky"
+      top={0}
+      direction="column"
+      bgColor="gray.100"
+      alignItems="center"
+      justifyContent="space-between"
+      p={4}
+    >
+      <Flex
+        alignItems="center"
+        direction="column"
+        bgColor="white"
+        borderRadius="md"
+        cursor="pointer"
+        color="piperider.500"
+        px={4}
+        py={2}
+      >
+        <Icon as={FiDatabase} boxSize={6} />
+        <Text fontSize="sm">Data</Text>
+      </Flex>
+
+      <Box>
+        <Divider borderColor="black" my={4} />
+        <Flex
+          alignItems="center"
+          direction="column"
+          cursor="pointer"
+          p={2}
+          onClick={() => modal.onOpen()}
+        >
+          <Icon as={FiMessageSquare} boxSize={6} />
+          <Text fontSize="xs">Feedback</Text>
+        </Flex>
+      </Box>
+
+      <Modal {...modal} size="2xl">
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader>Send Feedback</ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <Textarea
+              placeholder="We are always improving and would love ot hear your thoughts!"
+              value={feedback}
+              onChange={(event) => {
+                event.preventDefault();
+                setFeedback(event.target.value);
+              }}
+            />
+
+            <Flex direction="column" gap={2} mt={4}>
+              <Heading size="md">Metadata</Heading>
+              <UnorderedList>
+                <ListItem>
+                  Version: {window.PIPERIDER_METADATA.version}
+                </ListItem>
+              </UnorderedList>
+            </Flex>
+          </ModalBody>
+
+          <ModalFooter>
+            <Button mr={3} onClick={modal.onClose}>
+              Close
+            </Button>
+            <Button
+              colorScheme="piperider"
+              isLoading={isSending}
+              onClick={async () => {
```

Comment:
I think it'll be a good idea to add this touchpoint to our Cypress e2e specs, e.g. add data attributes `[data-cy="<stuff_about_element_user_behavior"]`

P.S. Welcome back! :)